### PR TITLE
Changed "Try it Description" by removing "we are currently working on...", adding "+1 on github for feature implementation" and other CSS changes

### PR DESCRIPTION
**Pull request by vandyliu**
[AB#34108](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34108) [AB#34110](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34110)

# Description:
- moved blue arrow SVG down to align better with text
- made it so the Modal never has a scroll bar 
- changed text for some sensors that said "we are currently working on" to something like  "_________ is not implemented. If you would like to see it implemented, +1 the feature request on Github" which links to this issue https://github.com/microsoft/vscode-python-devicesimulator/issues/268
- added padding to try it description because some words were going too far to the right

New versions:
new changes
new changes

Old versions (**Notice blue arrow SVG and words flowing too far to the right**):
Old versions
Old versions

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- Tested on micro:bit and CPX simulator
- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/view/styles/ToolBar.css [hunk 1]
```diff
@@ -142,20 +141,19 @@
     opacity: 90%;
     font-size: 14px;
     word-wrap: break-word;
+    padding-right: 10px;
```
Comment (nasadigital) [inline src/view/styles/ToolBar.css:144]: Just FYI, if you are worried about alignment you could also do:
```
text-align: justify;
text-justify: inter-word;
```
I am not sure if that's recommend from a UI standpoint, though.

**Review by nasadigital — APPROVED** [review]
LGTM! Thanks for also sharing screenshots of the associated changes in the PRs 😄

File: src/view/styles/ToolBar.css (hunk 1 above)
Comment (vandyliu) [reply src/view/styles/ToolBar.css:144]: Ooo, thanks for the tip 😄

**Review by xnkevinnguyen — APPROVED** [review]
Nice work!

**Pull request merged**